### DISCUSSION TITLE
Specify folder's display names on theia.workspace file

### DISCUSSION THREAD

**Discussion by aghayanlena** [eclipse-theia/theia#14270] [General]
Hi, 

Could you please guide how I can specify the folder display name on the workspace file?  Specifying it like this does not change  the display text:
```
{
    "folders": [
        {
            "path": "relative/path/to/folder",
            "name": "My Workspace Folder"
        }
    ]
}
``` 

Thanks,
Lena

**Comment by msujew** [comment]
Hey @aghayanlena,

I believe that the current implementation completely disregards whatever is in the file except for the `path` property. You will need to take a look at the implementation of the [workspace package](https://github.com/eclipse-theia/theia/tree/master/packages/workspace) to see what you have to adjust.
